10_Examples/jett-onboarding/src/hello_world.cpp

```diff
@@ -0,0 +1,14 @@
+#include "talking.h"
+#include <iostream>
+#include "string.h"
+
+using namespace std;
+
+int test();
```

Comment:
I assume this is an artifact from before migrating to separate test file. Thats no problem really, but good opportunity for learning stuff.

In this case, we have "declared" a function, but have not yet "defined" it. Basically, you could try to call this test function in your main, and this declaration is like a promise to the rest of the code that we will define it later. I bring it up because if you DONT define it later, you will get confusing errors that might not happen until you try to run it (i.e. it might compile fine, depends on situation).